### DISCUSSION TITLE
Multiple llm settings with own model and system prompt

### DISCUSSION THREAD

**Discussion by ninadkochkar** [Taewan-P/gpt_mobile#105] [Ideas]
Hi, 
I'd like to propose a feature that would enable us to create multiple copies of LLM settings, each with a unique name. This would give us the flexibility to configure different models (like GPT-4 and GPT-4-mini) with their own specific system prompts. By doing so, we could easily switch between these models as needed.

**Comment by Taewan-P** [comment]
This would be a similar request: #58, #95

**Comment by Taewan-P** [comment]
Hello, this feature is now implemented in [the latest version](https://github.com/Taewan-P/gpt_mobile/releases). Thanks!
